site/src/Command/CreateNotificationCommand.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\Command;
+
+use App\Util\Notification;
+use Doctrine\Common\Collections\ArrayCollection;
+use FOS\UserBundle\Model\UserManagerInterface;
+use OldSound\RabbitMqBundle\RabbitMq\ProducerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+
+/**
+ * Creates a Notification and publishes it to RabbitMQ
+ */
+class CreateNotificationCommand extends Command
+{
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'create:notification';
+
+    /**
+     * @var ProducerInterface
+     */
+    private $producer;
+
+    /**
+     * @var Notification
+     */
+    private $notification;
+
+    /**
+     * @var UserManagerInterface
+     */
+    private $userManager;
+
+    /**
+     * CreateNotificationCommand constructor.
+     *
+     * @param ProducerInterface    $notificationProducer
+     * @param Notification         $notification
+     * @param UserManagerInterface $userManager
+     */
+    public function __construct(ProducerInterface $notificationProducer, Notification $notification, UserManagerInterface $userManager)
+    {
+        parent::__construct();
+        $this->producer = $notificationProducer;
+        $this->notification = $notification;
```

Comment:
Why is `notification` an object member variable? It should be a local variable in the `execute()` method.